### PR TITLE
Add testing steps for email-alert-monitor to docs/ folder

### PR DESCRIPTION
**Pull request by mobaig**

### CODE REVIEW

File: docs/testing-email-alert-monitor.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Testing the Email Alert Monitor
+
+0. GOVUK Infrastructure have access to the relevant Google inboxes, the
+   Production Jenkins instance where the monitor is run from, and Icinga. All
+   three are required to test the monitor is working. Find someone from
+   Infrastructure before proceeding!
+0. In the inbox for whichever alert type you're checking, delete an email that
+   was received within during the period the monitor inspects (at time of
+   writing this is between '2 days ago' to '0.hour ago').
```
Comment (gpeng) [inline docs/testing-email-alert-monitor.md]: maybe...

> delete the email received during the inspection period

would read more betterer 😁

File: docs/testing-email-alert-monitor.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+0. In Jenkins, run the EmailAlertCheck task. This should fail and output an
+   error message reporting the absence of the email alert you deleted.
```
Comment (gpeng) [inline docs/testing-email-alert-monitor.md]: > of the deleted email.

**Comment by gpeng** [comment]
Apart from my couple of picky comments... Looks good.

File: docs/testing-email-alert-monitor.md (hunk 1 above)
Comment (mobaig) [inline docs/testing-email-alert-monitor.md]: Good lord that's sloppy. Updated!

**Comment by mobaig** [comment]
thanks @gpeng - comments addressed.

**Comment by gpeng** [comment]
:+1:

**Pull request merged**